package.json

```
@@ -6,7 +6,7 @@
     "build": "lerna run --scope @openmsupply-client/* build",
     "serve": "lerna run --scope @openmsupply-client/* --parallel serve",
     "clean": "lerna run --scope @openmsupply-client/* --parallel clean",
-    "compile": "lerna run --scope @openmsupply-client/* --parallel tsc",
+    "compile": "lerna run --scope @openmsupply-client/* --parallel tsc --since HEAD~1",
```

```suggestion
    "compile": "lerna run --scope @openmsupply-client/* --parallel tsc --since HEAD",
```

I believe it should be since `HEAD` rather than the first ancestor.
To check - I made a change in `Host.tsx` and running with HEAD only, then only the host package is compiled.
With the ancestor level then it compiles four packages (dashboard, requisitions, host, system). These are compiled even if I revert my changes.